src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -758,6 +758,18 @@
     <value>Cursor shape</value>
     <comment>Header for a control to select the shape of the text cursor.</comment>
   </data>
+  <data name="Profile_AdjustIndistinguishableColorsNever.Content" xml:space="preserve">
+    <value>Never</value>
+    <comment>An option to choose from for the "adjust indistinguishable colors" setting. When selected, we will never adjust the text colors.</comment>
+  </data>
+  <data name="Profile_AdjustIndistinguishableColorsIndexed.Content" xml:space="preserve">
+    <value>Indexed</value>
+    <comment>An option to choose from for the "adjust indistinguishable colors" setting. When selected, we will adjust the text colors for visibility only when the colors are part of this profile's color scheme's color table.</comment>
+  </data>

```

Comment:
```suggestion
  <data name="Profile_AdjustIndistinguishableColorsIndexed.Content" xml:space="preserve">
    <value>Only when the colors are a part of the color scheme</value>
    <comment>An option to choose from for the "adjust indistinguishable colors" setting.</comment>
  </data>
```
Similar comment to the other one. The comment is more useful than the displayed text itself, when it should be the other way around. This one in particular we should definitely rename because I don't know what "Indexed" means if I'm just poking around the settings.

Reply:
How about "Only for colors in the color scheme"?

Reply:
Good catch. I like that.

CC @cinnamon-msft in case she wants any word-smithing (but yeah let's go with this unless anybody disagrees)